File: Python/Program/PowerDevice.py
class PowerDevice:
    def __init__(self, wattage=0, efficiency_rating="", kecepatan_baca=0, kecepatan_tulis=0):
        self.wattage = wattage
        self.efficiency_rating = efficiency_rating
        self.kecepatan_baca = kecepatan_baca
        self.kecepatan_tulis = kecepatan_tulis
    
    def set_wattage(self, wattage):
        self.wattage = wattage
    
    def set_efficiency_rating(self, efficiency_rating):
        self.efficiency_rating = efficiency_rating
    
    def set_kecepatan_baca(self, kecepatan_baca):
        self.kecepatan_baca = kecepatan_baca
    
    def set_kecepatan_tulis(self, kecepatan_tulis):
        self.kecepatan_tulis = kecepatan_tulis
    
    def get_wattage(self):
        return self.wattage
    
    def get_efficiency_rating(self):
        return self.efficiency_rating
    
    def get_kecepatan_baca(self):
        return self.kecepatan_baca
    
    def get_kecepatan_tulis(self):
        return self.kecepatan_tulis
    
    def get_power_info(self):
        print(f"Wattage: {self.wattage} W, Efficiency Rating: {self.efficiency_rating}, Kecepatan Baca: {self.kecepatan_baca} MB/s, Kecepatan Tulis: {self.kecepatan_tulis} MB/s")

File: Java/Program/PowerDevice.java
public class PowerDevice {
    private int wattage;
    private String efficiencyRating;
    private int kecepatanBaca;
    private int kecepatanTulis;

    public PowerDevice() {
    }

    public PowerDevice(int wattage, String efficiencyRating, int kecepatanBaca, int kecepatanTulis) {
        this.wattage = wattage;
        this.efficiencyRating = efficiencyRating;
        this.kecepatanBaca = kecepatanBaca;
        this.kecepatanTulis = kecepatanTulis;
    }

    public void setWattage(int wattage) {
        this.wattage = wattage;
    }

    public void setEfficiencyRating(String efficiencyRating) {
        this.efficiencyRating = efficiencyRating;
    }

    public void setKecepatanBaca(int kecepatanBaca) {
        this.kecepatanBaca = kecepatanBaca;
    }

    public void setKecepatanTulis(int kecepatanTulis) {
        this.kecepatanTulis = kecepatanTulis;
    }

    public int getWattage() {
        return wattage;
    }

    public String getEfficiencyRating() {
        return efficiencyRating;
    }

    public int getKecepatanBaca() {
        return kecepatanBaca;
    }

    public int getKecepatanTulis() {
        return kecepatanTulis;
    }

    public void getPowerInfo() {
        System.out.println("Wattage: " + wattage + " W, Efficiency Rating: " + efficiencyRating + ", Kecepatan Baca: " + kecepatanBaca + " MB/s, Kecepatan Tulis: " + kecepatanTulis + " MB/s");
    }
}